### PR TITLE
SAK-41085 @deprecated Time => instant

### PR DESCRIPTION
**Pull request by axxter99**
MessageHeader: Instant getInstant();

MessageHeaderEdit: setInstant(Instant instant);

### CODE REVIEW

**Comment by axxter99** [comment]
@buckett ?

File: message/message-util/util/src/java/org/sakaiproject/message/util/BaseMessage.java [hunk 1]
```diff
@@ -3916,6 +3916,12 @@ public Time getDate()
 
 		} // getDate
 		
+
+		@Override
+		public Instant getInstant() {
+			Instant i = Instant.ofEpochMilli(m_date.getTime());
+			return i;
+		}
 		/**
```
Comment (adrianfish) [inline message/message-util/util/src/java/org/sakaiproject/message/util/BaseMessage.java]: Why not just return it without the variable?

File: message/message-util/util/src/java/org/sakaiproject/message/util/BaseMessage.java [hunk 2]
```diff
@@ -4164,6 +4170,13 @@ public void setDate(Time date)
 
 		} // setDate
 		
+
+		@Override
+		public void setInstant(Instant instant) {
+			Time time = m_timeService.newTime(instant.toEpochMilli());
+			setDate(time);
+		}
+
```
Comment (adrianfish) [inline message/message-util/util/src/java/org/sakaiproject/message/util/BaseMessage.java]: You could just do it one line. Small thing, I know, but still ...

File: message/message-util/util/src/java/org/sakaiproject/message/util/BaseMessage.java (hunk 1 above)
Comment (axxter99) [reply message/message-util/util/src/java/org/sakaiproject/message/util/BaseMessage.java]: OK

File: message/message-util/util/src/java/org/sakaiproject/message/util/BaseMessage.java (hunk 2 above)
Comment (axxter99) [reply message/message-util/util/src/java/org/sakaiproject/message/util/BaseMessage.java]: OK

**Comment by axxter99** [comment]
@adrianfish cheer! :)

**Pull request merged**